src/main/java/com/gooddata/retry/FailOverSettings.java

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright (C) 2007-2017, GoodData(R) Corporation. All rights reserved.
```

Comment:
shouldn't we update year in the template?

Reply:
Fixed